src/core/sys/posix/semaphore.d

```diff
@@ -128,27 +128,4 @@ version( Posix )
 int sem_timedwait(sem_t*, in timespec*);
 */
 
-version( linux )
-{
-    int sem_timedwait(sem_t*, in timespec*);
-}
-else version( OSX )
-{
-    int sem_timedwait(sem_t*, in timespec*);
-}
-else version( FreeBSD )
-{
-    int sem_timedwait(sem_t*, in timespec*);
-}
-else version (Solaris)
-{
-    int sem_timedwait(sem_t*, in timespec*);
-}
-else version( Android )
-{
-    int sem_timedwait(sem_t*, in timespec*);
-}
-else
-{
-    static assert(false, "Unsupported platform");
-}
+int sem_timedwait(sem_t*, in timespec*);
```

Comment:
[Defined this way in POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/functions/sem_timedwait.html).